packages/ens-utils/src/price.ts

```diff
@@ -123,14 +128,44 @@ export const multiplyPriceByNumber = (price1: Price, price2: number): Price => {
   };
 };
 
+export const CurrencySymbology = {
```

Comment:
This idea doesn't belong inside `ens-utils`. It should move into `CurrencySymbol` in `namekit-react`.

Reply:
@lightwalker-eth maybe what we need is to rename it?

Because the same symbology rules are used in formattedPrice, an ens-utils feature